llvm/include/llvm/CodeGen/MachineBasicBlock.h

```diff
@@ -846,8 +846,10 @@ class MachineBasicBlock
 
   /// Return the first instruction in MBB after I that is not a PHI, label or
   /// debug.  This is the correct point to insert copies at the beginning of a
-  /// basic block.
-  iterator SkipPHIsLabelsAndDebug(iterator I, bool SkipPseudoOp = true);
+  /// basic block. \p Reg is the register being defined for a spill/split during
```

Comment:
Actually spill does not define a register, it kills it. Restore defines register.